BlockServer/config/xml_converter.py

```diff
@@ -167,6 +168,9 @@ def meta_to_xml(data):
             edit = ElementTree.SubElement(edits_xml, TAG_EDIT)
             edit.text = e
 
+        protect_xml = ElementTree.SubElement(root, TAG_PROTECTED)
+        protect_xml.text = repr(data.isProtected).lower()
```

Comment:
Why are we using `repr` as opposed to `str`? Most of the time we should be using `str` (or one of the `six` string types if required)

Reply:
There were no specific reasons as to why we used `repr` as opposed to `str`.